source/_includes/links.md

```diff
@@ -30,10 +30,13 @@
 [openannotypes]: http://www.openannotation.org/spec/core/core.html#BodyTargetType
 [paging-prezi30]: {{ site.url }}{{ site.baseurl }}/api/presentation/3.0/#paging-properties
 [prev-version]: {{ site.url }}{{ site.baseurl }}/api/presentation/2.0/
+[prezi-api]: {{site.url}}{{ site.baseurl }}/api/presentation/3.0/
```

Comment:
maybe `prezi-api30` or something along those lines?

Reply:
We should also take out prev-version then, I guess?